ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/CommitTxnHandler.java

```diff
@@ -128,6 +130,20 @@ private List<WriteEventInfo> getAllWriteEventInfo(Context withinContext) throws
               })));
   }
 
+  private List<WriteEventInfo> getAllWriteEventInfoExceptMV(List<WriteEventInfo> writeEventInfoList) {
```

Comment:
do we even need to store the MV write event and do the filter later as done here?

Reply:
thanks @harshal-16 for the explanation! Going forward, please try to address open questions before merging.